### PR TITLE
New omfit_classes conda package for the omfit_classes python package

### PR DESCRIPTION
**Pull request by smithsp**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Mention @orso82 @kalling

### CODE REVIEW

**Comment by orso82** [comment]
Indeed @dopplershift I'll be happy to be listed as a maintainer.

File: recipes/omfit_classes/meta.yaml
Comment (smithsp) [reply recipes/omfit_classes/meta.yaml:40]: Yes, the repo is private.  We are working on releasing it.

File: recipes/omfit_classes/meta.yaml
Comment (smithsp) [reply recipes/omfit_classes/meta.yaml]: Done.  Thanks for noticing.

**Review by dopplershift — APPROVED** [review]
Just waiting on confirmation from @kalling on their willingness to be a maintainer (Or you can remove from this recipe to get it merged and add later in the feedstock)

**Comment by kalling** [comment]
@dopplershift Yes, I can be a maintainer.

**Comment by fsciortino** [comment]
This looks great -- I'm looking forward to being able to connect the `aurorafusion` conda recipe to `omfit_classes` to automate conda dependency builds. Thank you for working on this!

**Comment by smithsp** [comment]
Thanks @dopplershift .

**Pull request merged**
